pkg/migrations/op_set_default_test.go

```diff
@@ -248,7 +250,109 @@ func TestSetDefault(t *testing.T) {
 						&migrations.OpAlterColumn{
 							Table:   "users",
 							Column:  "name",
-							Default: ptr("NULL"),
+							Default: nullable.NewNullableWithValue("NULL"),
```

Comment:
This diff is very confusing. What I actually did was duplicate the test where we set the value to `NULL` with another example where we set it to `null`.

`Default: nullable.NewNullableWithValue("NULL")` vs `Default: nullable.NewNullNullable[string](),`